File: 8ball.js
// ask for username
let userName = "Danny";

// hello message
userName ? console.log(`Hello, ${userName}!`) : console.log("Hello!");

// ask user for question
const userQuestion = "Is this a random number";
console.log(`${userName} asked ${userQuestion}`);

// get random number
let randomNumber = Math.floor(Math.random() * 8);

// print out answer based on random number
let text = '';

switch (randomNumber) {
  case 0:
    text = "It is certain";
    break;
  case 1:
    text = "It is certain";
    break;
  case 2:
    text = "It is decidedly so";
    break;
  case 3:
    text = "Reply hazy try again";
    break;
  case 4:
    text = "Cannot predict now";
    break;
  case 5:
    text = "Do not count on it";
    break;
  case 6:
    text = "My sources say no";
    break;
  case 7:
    text = "Outlook not so good";
    break;
  case 8:
    text = "Signs point to yes";
    break;
}

console.log(`Answer: ${text}`)

File: 8ball.py
import random

name = "John"
question = "Nice weather today?"
answer = "" 

if name == "":
  print("Question: " + question)
else:
  print(name + " asked: " + question)

random_number = random.randint(1,9)

if random_number == 1:
  answer = "Yes - definitely"
elif random_number == 2:
  answer = "It is decidedly so"
elif random_number == 3:
  answer = "Without a doubt"
elif random_number == 4:
  answer = "Reply hazy, try again"
elif random_number == 5:
  answer = "Ask again later"
elif random_number == 6:
  answer = "Better not tell you now"
elif random_number == 7:
  answer = "Ask again later"
elif random_number == 8:
  answer = "Better not tell you now"
else: 
  answer = "Very doubtful"

print("Answer: " + answer)
